stac_fastapi/elasticsearch/utils.py

```diff
@@ -142,7 +150,7 @@ def get_queryset(client, table: "Document" , **kwargs) -> Search:
 
         field_mapping = {
             'datetime': 'properties.datetime',
-            'bbox': 'spatial.bbox.coordinates'
+            'bbox': 'bbox.coordinates'
```

Comment:
This will be the same as above. 
I agree that the setup with separate mappings could get a little messy.
You could create a separate package which defines a mapping using the [elasticseach_dsl document](https://elasticsearch-dsl.readthedocs.io/en/latest/persistence.html#document). Perhaps then use this in the elasticsearch output for the generators and in the test index creation. Either that or just keep the two YAML files up to date and in sync

Reply:
The test data can be changed to reflect the mapping changes.